package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "mini-project",
```

Comment:
can we use a better name for creating projects, this should be future proof and allow naming conventions for multiple projects in the same app